xml/art_sle_ha_nfs_quick.xml

```diff
@@ -454,6 +454,9 @@ include /etc/drbd.d/*.res;</screen>
    <para>
     To configure these resources, run the following commands from the
    &crmshell;:
+   <remark>taroth 2018-03-05: ygao, should 'master-max' and 'master-node-max' be
+    replaced with 'promoted-max' and 'promoted-node-max', respectively? 
```

Comment:
Hrm, good question. Although  the old names are now deprecated in pacemaker, the new names are not explicitly  promoted in crmsh yet.

@krig Do you think we should simply list the "promotable", "promoted-max" and "promoted-node-max" as supported meta attributes for clone type, and start to change some configuration snippets in the documents like this, or we wait for SLE15SP1?